validator/src/main/java/io/avaje/validation/Validator.java

```diff
@@ -12,12 +13,13 @@
 import java.util.Map;
 import java.util.ResourceBundle;
 import java.util.ServiceLoader;
+import java.util.Set;
 import java.util.function.Supplier;
 
 public interface Validator {
 
   /** Validate the object using the default locale. */
-  void validate(Object any) throws ConstraintViolationException;
+  void validate(Object value, @Nullable Class<?>... groups) throws ConstraintViolationException;
```

Comment:
With varargs groups I don't think it's nullable ?

Reply:
If you don't pass anything for varargs it's null